site/en/docs/extensions/mv3/tut_analytics/index.md

```diff
@@ -1,166 +1,279 @@
+This tutorial demonstrates how to track the usage of your extension using Google Analytics. You can find a [working Google Analytics 4 sample on Github][1], where [google-analytics.js][2] includes all the Google Analytics related code.
 
 ## Requirements {: #toc-requirements }
 
-This tutorial expects that you have some familiarity writing extensions for Google Chrome. If you
-need information on how to write an extension, please read the [Getting Started tutorial][2].
+This tutorial assumes you are familiar with writing Chrome extensions. If you need information on how to write an extension, please read the [Getting Started tutorial][3].
 
-You will also need a [Google Analytics account][3] set up to track your extension. Note that when
-setting up the account, you can use any value in the Website's URL field, as your extension will not
-have an URL of its own.
+You must also set up a [Google Analytics 4 account][4] to track your extension. Note that when setting up the account, you can use any value in the Website's URL field, as your extension will not have an URL of its own.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/0X6Knat4ozKRyF15shTn.png",
-       alt="The analytics setup with info for a chrome extension filled out", height="82", width="400" %}
+## Using the Google Analytics Measurement Protocol {: #measurement-protocol }
 
-## Installing the tracking code {: #toc-installing }
+Since Manifest V3, Chrome Extensions are [not allowed to execute remote hosted code][5]. This means you have to use the [Google Analytics Measurement Protocol][6] for tracking extension events. The Measurement Protocol lets you send events directly to Google Analytics servers via HTTP requests. A benefit of this approach is that it lets you send analytics events from everywhere in your extension, including your service worker.
 
-The standard Google Analytics tracking code snippet fetches a file named `ga.js` from an SSL
-protected URL if the current page was loaded using the `https://` protocol.
+{% Aside 'caution' %}
+Using the Measurement Protocol means that some information, such as geolocation, will not be included.
+{% endAside %}
 
-**Chrome extensions may _only_ use the SSL-protected version of `ga.js`**. Loading `ga.js` over insecure
-HTTP is disallowed by Chrome's default [Content Security Policy][4]. This, plus the fact that Chrome
-extensions are hosted under the `chrome-extension://` schema, requires a slight modification to the
-usual tracking snippet to pull `ga.js` directly from `https://ssl.google-analytics.com/ga.js`
-instead of the default location.
+### Set up API credentials {: #setup-credentials }
 
-Below is a modified snippet for the [asynchronous tracking API][5] (the modified line is bolded):
+The first step is to obtain an `api_secret` and `measurement_id`. Follow the [Measurement Protocol documentation][6] for how to get these for your Analytics Account.
```

Comment:
NOT Is there a more specific link on obtaining an `api_secret` and `measurement_id`?

Reply:
yes, added